oci/utils_unix_test.go

```diff
@@ -0,0 +1,44 @@
+//go:build !windows && !darwin
+// +build !windows,!darwin
```

Comment:
I excluded the test on darwin for now as it's failing there, but wondering if `HostDevices()` should be supported on macOS, or if we need a stub for it?